src/index.html

```diff
@@ -3,12 +3,53 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <a class="icon" href="#home">
+        <img src="./images/logo.png" alt="img">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="apple" class="Nav__link is-active">APPLE</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="samsung">samsung</a>
+          </li>
+          <li class="nav__item">
```

Comment:
I suggest you putting 1 blank line between sibling tags (when they have some content) to improve readability. But not parent and its child
```suggestion
    <header class="header">
      <a class="icon" href="#home">
        <img src="./images/logo.png" alt="img">
      </a>

      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="apple" class="Nav__link is-active">APPLE</a>
          </li>

          <li class="nav__item">
            <a class="nav__link" href="samsung">samsung</a>
          </li>

          <li class="nav__item">
```